dao/mongodb.rya/src/main/java/org/apache/rya/mongodb/dao/SimpleMongoDBStorageStrategy.java

```diff
@@ -96,12 +103,19 @@ public DBObject getQuery(final RyaStatement stmt) {
 
     @Override
     public RyaStatement deserializeDBObject(final DBObject queryResult) {
-        final Map result = queryResult.toMap();
+        final Map<?, ?> result = queryResult.toMap();
```

Comment:
A Map<String, Object> is returned here but I would need to add @SuppressWarnings("unchecked") here which I'd prefer not to do.  Leaving it as just Map generates a raw type warning.  Map<?, ?> gets rid of the warning and is safe.